frontend/elements/src/entry/home/widgets/studentcode-section.ts

```diff
@@ -49,25 +49,25 @@ import { MEDIA_UI } from '@utils/path';
     `];
   }
   @property()
-  kindimage:Image = ""; 
+  kindimage:Image = "play"; 
 
 
   render() {
     const {kindimage } = this;
 
-    const path = kindimage === "play" ? " Illustration_JIG_Sad_1.png"
+    const path = kindimage === "play" ? "Image_JIG_StudentCode%402x.png"
         : kindimage === "askforhelp" ? "Image_Jig_Studentcode_error2@2x.png"
         : kindimage ==="tryagain" ? "Image_Jig_Studentcode_error1@2x.png"
-        : "";
+        :nothing;
```

Comment:
`nothing` isn't a valid path for `img-ui` - in that case just hide the `img-ui` completely